environment.yml

```diff
@@ -15,6 +15,7 @@ dependencies:
   - iverilog
   - libxml2=2.9.10=20200629_180127
   - icestorm
+  - prjxray-db
```

Comment:
This should be pinned, not floating

Reply:
I'm not sure we should be using the package here?  At least not in this PR?  Maybe continue to use the submodule, and make a future PR?

Reply:
As commented earlier, IMO it would be better to test this as well at least for the install_test that checks whether everything is all right with the toolchain and the toolchain wrappers.